lib/element.js

```diff
@@ -353,7 +353,29 @@ ElementArrayFinder.prototype.toElementFinder_ = function() {
  */
 ElementArrayFinder.prototype.count = function() {
   return this.getWebElements().then(function(arr) {
-    return arr.length;
+    var list = [];
+    arr.forEach(function(webElem) {
```

Comment:
A `forEach` that ends with a list `push` is just a `map` call?   Structuring this as `list = arr.map(function(webElem) { ... })` might be cleaner.